### PR TITLE
Invitation. Close #13

### LINKED ISSUE #13: Add invitation feature
## Requirements
- A user who create a room will be the owner of the room
  - So there is one owner per one room
- An Owner can invite anyone to the room
- An invited user automatically joins the room

### PR DESCRIPTION
**Pull request by fgro93**
#13

### CODE REVIEW

File: client/simpleChat.js [hunk 1]
```diff
@@ -14,6 +14,20 @@
...
+    Template.users.helpers({
+      users: function() {
+        return Meteor.users.find();
+      }
+    });
+
+    Template.body.helpers({
+      isOwner: function() {
+        let currentRoomName = Session.get(sessionKeyRoomName);
+        let currentRoom = Rooms.findOne({roomName: currentRoomName});
```
Comment (ss0201) [inline client/simpleChat.js]: oh i didn't notice in previous pull request, but it's better to use `const` for non-changing values. by this, the meteor compiler warns you if you mistakingly try to re-assign to a const value.

File: client/simpleChat.js [hunk 2]
```diff
@@ -38,6 +55,17 @@
       }
     }
 
+    Template.users.events = {
+      'click .user': function(event){
+        let currentRoomName = Session.get(sessionKeyRoomName);
+        let currentRoom = Rooms.findOne({roomName: currentRoomName});
+        if(currentRoom){
+          currentRoom.invitedUsers.push(event.target.innerText);
```
Comment (ss0201) [inline client/simpleChat.js]: there should be a condition to prevent the same user to be invited multiple times

**Comment by ss0201** [comment]
oops, you seem to have closed the issue, but i still have some comments on your PR. anyway i'll create another issue for those.

**Comment by fgro93** [comment]
My fault. I already commited a few more things on that branch and didn't want to deal with the merge conflicts. All `let` are already changed to 'const' where possible. And I will add the condition in a few minutes.

Starting from now, I will wait till I get  **LGTM**. Won't happen again. Sorry

**Comment by ss0201** [comment]
it's ok bro :)

**Pull request merged**